editor/package.json

```diff
@@ -145,6 +145,7 @@
     "platform-detect": "3.0.0",
     "prettier": "2.0.5",
     "prop-types": "15.6.2",
+    "pubsub-js": "^1.9.2",
```

Comment:
```suggestion
    "pubsub-js": "1.9.2",
```

Reply:
I am the worst. is there a way to configure npm to avoid ^s by default?